### PR TITLE
docs/fix-formatter

### PR DESCRIPTION
**Pull request by bre1470**
Fixed formatter types and doclets. Tooltip types should show up in class references.

### CODE REVIEW

File: ts/parts/Tooltip.ts [hunk 1]
```diff
@@ -247,6 +247,8 @@ declare global {
  * @typedef {"callout"|"circle"|"square"} Highcharts.TooltipShapeValue
  */
 
+''; // separates doclets above from variables below
```
Comment (pawelfus) [inline ts/parts/Tooltip.ts]: ?

Comment (bre1470) [reply ts/parts/Tooltip.ts]: Yeah. That was interesting. All the doclets above where attached to the variable below, which was internal and filtered out. Therefor no interface was going into the class reference. :)

Comment (jon-a-nygaard) [reply ts/parts/Tooltip.ts]: As I understand it is a hack to make sure that the doclets is picked up by JSDoc without the variables below interfering with the result.

Maybe you can confirm or correct me on this @bre1470?

Comment (pawelfus) [reply ts/parts/Tooltip.ts]: I guess simple:

// ---- doclets separator ----

Won't work? :) Maybe it's better to place doclets at the end of the file? And make this a general rule? I'm worried that in future someone else will spot the same issue, wasting time to find answer "whyyyyy?"

**Pull request merged**